apps/www/_blog/2022-08-17-supabase-soc2.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Supabase is SOC2 compliant'
+description: Supabase is SOC2 Type 1 compliant. We received an unqualified certification from our auditors.
+author: inian
+image: lw5-soc2/thumb.jpg
+thumb: lw5-soc2/thumb.jpg
+tags:
+  - launch-week
+date: '2022-08-17'
+toc_depth: 3
+---
+
+Supabase is now SOC2 Type 1 compliant. Let’s dig into what that means, and explain the process we went through to get there. If you’re building a SaaS product, this blog post should provide a rough guide to getting your SOC2 certification.
+
+## What is SOC2?
+
+SOC2 is a compliance standard developed by the American Institute of CPAs. Yup CPAs - a bunch of accountants came up with the generally accepted compliance security standard for SaaS companies. SOC2 evaluates companies on 5 Trust Services Criteria: security, availability, processing integrity, confidentiality, and privacy. There are two types of compliance, both requiring an external audit:
+
+- Type 1: checks if a company is compliant at a point in time.
+- Type 2: checks if a company stays compliant during an observation window (typically 6 months to a year).
+
+## Why did we get SOC2 certified?
+
+The main reason to go through a SOC2 audit is a simple one - customers expect it.
+
+The Supabase team was spending a considerable amount of time filling out security questionnaires and fielding calls with potential customers - walking them through our security practices and how we protect customer data.
+
+We are a database company — customers trust us with their data. We started with a SOC2 audit since one of the main things covered is our handling of sensitive customer data.
+
+It is also much easier to become-and remain-SOC2 compliant when you institute the appropriate practices and standards as early as possible.
+
+## Choosing a compliance monitoring tool
+
+There are a few tools that plug into your cloud providers, task management system, identity, and HR providers and make it easy to monitor, collect and submit evidence to auditors. We evaluated [Vanta](https://www.vanta.com/), [Drata](https://drata.com/), and [Secureframe](https://secureframe.com/). Our stack is pretty standard (AWS, GCP, GSuite, and Github). All the tools we evaluated were integrated with these systems.
+
+Some of them had overly broad permissions when integrating with our vendors (read/write access to Github for example) which made them a no-go. Some came with a lightweight agent that we could use for Mobile Device Management (MDM). Other than that, the products are largely the same.
+
+We chose Vanta, with Drata coming in a close second. Vanta supported more compliance standards at that time and is a fellow YC company. The gap between these tools is much closer now since all of them support the compliance standards we have our eyes on.
```

Comment:
```suggestion
We chose Vanta, with Drata coming in a close second. Vanta supported more compliance standards (at that time) and is a fellow YC company (which sealed the deal). The gap between these tools is much closer now since all of them support the compliance standards we have our eyes on.
```